### DISCUSSION TITLE
In react-redux8.0.1, what type do I need to give to make my Dispatch error free

### DISCUSSION THREAD

**Discussion by Silence-dream** [reduxjs/react-redux#1913] [Q&A] [answered]
I've implemented a minimal example [here](https://codesandbox.io/s/blazing-surf-22kiwj)

![image](https://user-images.githubusercontent.com/42824008/165666055-3b250891-10cc-4f24-8bc7-76246180cd22.png)

I want to make that paragraph disappear, but I don't know how to write

What is the proper way to use useDispatch()?

**Comment by markerikson** [comment, marked as answer]
Follow the TS usage guidelines shown in our docs for inferring the type of `dispatch` from your store, and using pre-typed hooks.  Also note that the store setup works best if you're using Redux Toolkit's `configureStore` (which you should be using anyway):

https://react-redux.js.org/using-react-redux/usage-with-typescript#define-root-state-and-dispatch-types
